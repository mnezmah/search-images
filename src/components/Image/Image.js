import React from 'react'
import './Image.scss'

export default function Image(props) {
  return (
    <div className='image-container'>
      <img
        className='image'
        src={props.url}
        alt='' />
    </div>
  )
}