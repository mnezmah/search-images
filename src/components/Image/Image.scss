@import "../../Styling/main.scss";

.image-container {


  width: 77vw;
  height: 26vh;

  //Desktop
  // position: relative;
  // display: inline-block;
  // vertical-align: middle;
  // // width: 33vh;
  // 

  .image {
    position: relative;
    border-radius: 13px;
    border: 3px solid $main-white;
    width: 100%;
    height: 100%;
    max-height: 100%;
    min-width: 100%;
    // filter: drop-shadow(30px 10px 4px #4444dd);
  }
}
