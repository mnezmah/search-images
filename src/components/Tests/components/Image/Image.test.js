import React from 'react'
import { shallow } from 'enzyme'
import Image from '../../../Image/Image'

test('should render Image corectly', () => {
  const wrapper = shallow(<Image />)
  expect(wrapper).toMatchSnapshot()
})