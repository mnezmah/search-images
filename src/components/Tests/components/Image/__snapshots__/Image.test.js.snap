// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Image corectly 1`] = `ShallowWrapper {}`;
